packages/flutter/test/material/icon_button_test.dart

```diff
@@ -337,6 +337,39 @@ void main() {
     await gesture.up();
   });
 
+  testWidgets('IconButton with explicit splash radius',
+      (WidgetTester tester) async {
+    const double splashRadius = 30.0;
+
+    final Widget buttonWidget = wrap(
```

Comment:
The structure of this test does match most of the others in this file however, I think it would preferable to establish a new slightly simpler pattern. 

```dart
await tester.pumpWidget(
  MaterialApp(
    home: Material(
      child: Center(
        IconButton(
          icon: const Icon(Icons.android),
          splashRadius: splashRadius,
          onPressed: () { /* enable the button */ },
        ),
      ),
    ),
  ),
);
```
There are one or two tests that have already been set up like this.

Reply:
Done. Thanks for the pointer.